### ISSUE TITLE
--recursive makes mochify ignore `entry`

### ISSUE THREAD

**Issue by cjohansen** [mantoni/mochify.js#120]
Adding the `--recursive` option makes `mochify` ignore my file paths and insist on the default one:

```sh
✗ ./node_modules/.bin/mochify --transform hintify --transform babelify client/test/*.js
# phantomjs:
  ......
  6 passing (42ms)

✗ ./node_modules/.bin/mochify --transform hintify --transform babelify client/test/*.js --recursive
Error: Nothing found for "./test/**/*.js".

✗ ./node_modules/.bin/mochify --transform hintify --transform babelify --recursive client/test/*.js
Error: Nothing found for "./test/**/*.js".
```

**Comment by mantoni** [comment]
Hacked up a fix and released in `2.14.2`. Thanks for reporting :beer:

**Comment by cjohansen** [comment]
:+1:
